managers/tests/test_views.py

```diff
@@ -342,3 +345,104 @@ def test_remove_task_activities_view_should_change_task_status_to_not_active(sel
         self.client.post(self.url)
 
         self.assertIn(self.active_task_activity, TaskActivityType.objects.filter(is_active=False))
+
+
+class ExpectedHoursBaseTests(TestCase):
+    def setUp(self):
+        super().setUp()
+        self.current_time = timezone.now()
```

Comment:
Try not to test something with current time because it might leads to not deterministic tests. If you want to use `timezone.now` please add `freeze_time` decorator on class.

Reply:
added